skill-tree/src/main/resources/dummy-data/endorsements/endorsements.json

```diff
@@ -0,0 +1,142 @@
+[
```

Comment:
why this is not inside dummy-data folder

Reply:
Currently it's under _resources/dummy-data/endorsements  _
Should I move it somewhere else? Basically create a dummy data folder outside resources/ ?

Reply:
moved it to resources/dummy-data-folder